src/components/editor-page/editor-pane/one-dark.scss

```diff
@@ -1,11 +1,7 @@
-/*
- * Copyright (c) 2015 Hikio - twitter.com/hik_io
- * SPDX-FileCopyrightText: 2015 Hikio - twitter.com/hik_io
- *
- * SPDX-License-Identifier: MIT
+/*!
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
  *
- * Atom One Dark Theme
- * GitHub https://github.com/hikio/brackets-one-dark
+ * SPDX-License-Identifier: AGPL-3.0-only
  */
 
 /*
```

Comment:
Wrong again as in clean-up-2 (https://github.com/hedgedoc/react-client/pull/987#discussion_r567330083)

Reply:
will automaticly be fixed when clean-up-2 is merged

Reply:
Rebased